test/Groups/subgroups_and_cosets.jl

```diff
@@ -181,9 +181,9 @@ end
       @test representative(rc) == x
       @test representative(lc) == x
       @test representative(dc) == x
-      @test Set(elements(rc)) == Set(z for z in rc)          # test iterator
-      @test Set(elements(rc)) == Set(h*x for h in H)
-      @test Set(elements(lc)) == Set(x*h for h in H)
+      @test Set(collect(rc)) == Set(z for z in rc)          # test iterator
```

Comment:
Is this test still relevant?

Reply:
Indeed, this now uses iteration on both sides, so it's somewhat pointless... I guess we could still check if e.g. length(collect(rc)) == length(rc) (assuming there is a length method for rc

Reply:
Replaced it.